### ISSUE TITLE
Question on Git Hub Actions

### ISSUE THREAD

**Issue by iranzo** [Pelican-Elegant/elegant#629]
Now that Git Hub Actions are available for most repos it might be good to move current TravisCI jobs to Git Hub Actions workflows.

I'm using it for blog-o-matic so that it can even publish to another branch without the need of token being provided as well as extra checks like pre-commit, etc

Would be interesting to get the change done here?

**Comment by talha131** [comment]
yeah sure. I too want it, but I could not explore Github actions due to lack of time.

**Comment by iranzo** [comment]
Check the https://github.com/iranzo/blog-o-matic/tree/master/.github folder for how I'm using it :-)

**Comment by talha131** [comment]
Cool! I will look in further details later.

**Comment by iranzo** [comment]
I've raised the following PR's:


(feat) Move yaspeller execution to pre-commit by iranzo · Pull Request #634 · Pelican-Elegant/elegant
https://github.com/Pelican-Elegant/elegant/pull/634

(feat) Adds precommit Git Hub Action by iranzo · Pull Request #633 · Pelican-Elegant/elegant
https://github.com/Pelican-Elegant/elegant/pull/633

(feat) Greetings on new issues/pr's by iranzo · Pull Request #632 · Pelican-Elegant/elegant
https://github.com/Pelican-Elegant/elegant/pull/632

(feat) Labeleler by iranzo · Pull Request #631 · Pelican-Elegant/elegant
https://github.com/Pelican-Elegant/elegant/pull/631

It will still be missing moving out of Travis-CI any other missing check (as the site rendering is done via netlify)

**Comment by iranzo** [comment]
This might be interesting:
- https://github.com/marketplace/actions/conventional-pr-title
